prior_noise:
    R_IB: [0.03, 0.03, 0.03]
    I_p_IB: [0.01, 0.01, 0.01]
    I_v_IB: [0.01, 0.01, 0.01]
    b_a: [2.0e-6, 2.0e-6, 2.0e-6]
    b_g: [1.7e-6, 1.7e-6, 1.7e-6]

noise:
    radar:
        doppler: 0.05 # sigma
        track: [0.2, 0.2, 0.2] # BearingRange3D
        loss: 5  # quadratic, 1: fair, 2: huber, 3: cauchy, 4: geman-mcclure, 5: welsch, 6: tukey
    baro:
        height: 0.2 # RMS/sqrt(dt) (BMP390 0.11m sigma, dt=0.02)
        loss: 0 # 0: quadratic, 1: fair, 2: huber, 3: cauchy, 4: geman-mcclure, 5: welsch, 6: tukey

radar:
    track_age: 10 # track for n frames, i.e., 0 = only current frame, 1 = current and previous frame, etc.

baro:
    active: False

imu:
    acc_sigma: 0.034596
    bias_acc_sigma: 5.33e-6
    initial_bias_acc: [0.0, 0.0, 0.0]

    gyro_sigma: 3.87e-4
    bias_omega_sigma: 1.66e-8
    initial_bias_gyro: [0.0, 0.0, 0.0]

    integration_sigma: 1.0e-5
    bias_acc_int_sigma: 1.0e-3
    bias_omega_int_sigma: 1.0e-3
    frequency: 200

isam2:
    relinearize_threshold: 0.1
    relinearize_skip: 10
    find_unused_factor_slots: False
    cache_linearized_factors: True
    enable_partial_relinarization_check: False
    smoother_lag: 5.0 # s WARNING: This time needs to be greater than any imaginable sensor arrival delay
    optimizer: 0 # 0: Gauss-Newton, 1: Dogleg
    gn:
        wildfire_threshold: 0.001

